common/travis/fetch_upstream.sh

```diff
@@ -9,4 +9,4 @@ elif command -v git >/dev/null 2>&1; then
 fi
 
 /bin/echo -e '\x1b[32mFetching upstream...\x1b[0m'
-$GIT_CMD fetch --depth 200 git://github.com/void-linux/void-packages.git master
+$GIT_CMD fetch --depth 200 git://github.com/oSoWoSo/void-packages.git master
```

Comment:
That's not how this works, rebase on top of our master, remove these CI changes and make sure to have only one commit with proper description.

Reply:
reverted